.gitpod/readme.md

```diff
@@ -45,7 +45,7 @@ where:
 ## Install Ignite CLI locally
 
 ```
-curl https://get.ignite.com/cli! | bash
+curl https://get.ignite.com/cli | bash
 ```
 
 The latest `ignite` binary is downloaded from the project repo and installed in `/usr/local/bin`. Learn more about [installing Ignite CLI](https://docs.ignite.com/guide/install.html).
```

Comment:
@aljo242 If you remove the trailing `!` from the curl command above, the binary will be download in the current directory, not in `/usr/local/bin`.

I'm not sure what's the best for the user. Personally, what I dislike with `!` is it forces the user to run the command as root, so ideally the command should be written as `sudo curl https://get.ignite.com/cli! | bash`, which is a terrible practice.

I would go without `!`, but then the following sentence should be modified :

```suggestion
The latest `ignite` binary is downloaded from the project repo into your current directory. Learn more about [installing Ignite CLI](https://docs.ignite.com/guide/install.html).
```

Reply:
Once we are agree on going with or without the trailing `!`, we should update all the curl commands.

Reply:
Tracking the issue [here](https://github.com/ignite/cli/issues/3041).  I wanted to see if we could get some more opinions in a standalone issue.  We can finalize this PR without finishing this bit!